.azure-pipelines.yml

```
@@ -155,7 +155,8 @@ jobs:
         ${AMReX_CMAKE_FLAGS}    \
         ${WARPX_CMAKE_FLAGS}    \
         -DWarpX_TEST_CLEANUP=ON \
-        -DWarpX_TEST_FPETRAP=ON
+        -DWarpX_TEST_FPETRAP=ON \
+        -DCMAKE_BUILD_TYPE=Debug
```

Try `RelWithDebInfo` for faster runtimes:
```suggestion
        -DCMAKE_BUILD_TYPE=RelWithDebInfo
```